minutes/_posts/2017-12-01-december-1-2017.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: contact
+---
+
+# Minutes of the 7th meeting of the Scala Center, Q4 2017
+
+Minutes are [archived](https://scala.epfl.ch/records.html) on the
+Scala Center website.
+
+## Summary
+
+The following agenda was distributed to attendees:
+[agenda](https://github.com/scalacenter/advisoryboard/blob/master/agendas/007-2017-q4.md).
+
+Scala Center engineering activities for the past quarter focused on
+Scala 2.13 collections, macros, Scalafix and Scalameta, Scala compiler
+profiling, Zinc, and Bloop.  Progress was made in other areas as well.
+
+Full details on these activities are in
+[Heather's report](https://scala.epfl.ch/minutes/ab-reports/2017-q4.html).
+
+Other topics discussed at the meeting included LSP (Language Server Protocol)
+support for Scala.
+
+No new proposals were submitted this quarter.
+
+## Date, Time and Location
+
+The meeting took place virtually, via UberConference, at 4:00pm
+(UTC) on Friday, December 1, 2017.
+
+Minutes were taken by Seth Tisue (secretary).
+
+## Attendees
+
+### Attendees present
+
+Board members:
+
+- Ross Baker, Verizon
+- James Belsey, Morgan Stanley
+- Thomas Gawlitza, SAP
+- Stu Hood, Twitter
+- Lars Hupel, community/Typelevel
```

Comment:
I would be tempted to leave "Typelevel" off, and likewise "Artima", just because (unlike the sponsors) they're not officially meant to be representing these organizations.

Reply:
done